src/storm/settings/modules/IOSettings.h

```diff
@@ -5,6 +5,7 @@
 
 #include "storm-config.h"
 #include "storm/builder/ExplorationOrder.h"
+#include "storm/environment/modelchecker/ModelCheckerEnvironment.h"
```

Comment:
i am not so happy with this include. Including the IOSettings now means including all environments. Maybe that is anyway already the case, so dont sweat it :-)